src/helper/filterLinks.tsx

```diff
@@ -0,0 +1,7 @@
+import { Filter } from '../types/Filter';
```

Comment:
Add a type for the `filterLinks` array to ensure type safety. For example, you can create a new type `FilterLink` and use it to define the structure of the objects within the array. 🌟